cmd/admission-webhook/main.go

```diff
@@ -15,29 +15,34 @@ package main
 
 import (
 	"flag"
+	"github.com/openshift/generic-admission-server/pkg/apiserver"
```

Comment:
Could you configure your editor to use `goimports` to format code? This will ensure the `import` package order consistent.

Reply:
updated and add `goimport` into FileWatcher. Missing it was a mistake.